src/common/Section/Section.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+
+export const Section = styled.section`
+	margin: 56px 0 0 0;
+
+	@media (max-width: 767px) {
+		margin: 24px 0 0 0;
+	}
+`
+
+export const SectionTitle = styled.h2`
+	margin: 0 0 32px 0;
+	font-size: 36px;
+	font-weight: 600;
+
+	@media (max-width: 767px) {
```

Comment:
I would suggest to use references to theme.js (breakpoints: mobile) instead of @media (max-width: 767px).